packages/app/pages/api/auth/[...nextauth].ts

```diff
@@ -0,0 +1,67 @@
+import EarthdataLoginProvider from 'auth/providers/earthdata-login'
+import log from 'lib/log'
+import NextAuth, { AuthOptions } from 'next-auth'
+import { getUsersDb } from 'auth/db'
+
+export const authOptions: AuthOptions = {
+    // use our mEditor logger for NextAuth log messages
+    debug: log.level === 'debug',
+    logger: {
+        error(code, metadata) {
+            log.error(code, metadata)
+        },
+        warn(code) {
+            log.warn(code)
+        },
+        debug(code, metadata) {
+            log.debug(code, metadata)
+        },
+    },
+
+    // use JWT tokens for tracking the user's session
+    session: {
+        strategy: 'jwt',
+    },
+
+    // custom page overrides
+    pages: {
+        signIn: '/meditor/signin',
+    },
+
+    // configure one or more authentication providers
+    providers: [
```

Comment:
Here's where we can add more providers as we go (Github, Google, Launchpad, etc.)